js/pages/prediction/components/prediction-executions.js

```diff
@@ -87,6 +96,26 @@ define([
 			this.isResultsDownloading = ko.computed(() => this.downloading().length > 0);
 			this.executionGroups = ko.observableArray([]);
 			this.isViewGenerationsPermitted() && this.startPolling();
+			this.loadData();
+		}
+
+		highlightGeneration() {
```

Comment:
Looks like code duplication (see `highlightGeneration` above)

Reply:
fixed. Moved to dataTableUtils